### DISCUSSION TITLE
【2022.12.14】community meeting notes

### DISCUSSION THREAD

**Discussion by Stevent-fei** [sealerio/sealer#1916] [General]
hi all, 今晚8点，有咱们sealer的开发者会议，大家有什么议题可以提前准备哦。

**Comment by starnop** [comment]
add image spec definition  https://github.com/sealerio/sealer/pull/1915

**Comment by starnop** [comment]
New design for basefs https://github.com/sealerio/sealer/pull/1909

**Comment by kakaZhou719** [comment]
sealer new doc https://github.com/sealerio/sealer.cool/issues/15

**Comment by YTGhost** [comment]
More detail about How to implement image policy for sealer using Kyverno https://github.com/sealerio/sealer/issues/1899

**Comment by zhy76** [comment]
sealer CI e2e-test by using github action #1633
